### PR TITLE
Update configure-atp-anti-phishing-policies.md

### PR DESCRIPTION
**Pull request by urgandhi**
Added notes for the limits supported for users and domains.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/configure-atp-anti-phishing-policies.md [hunk 1]
```diff
@@ -166,12 +166,14 @@ Use the following procedures to modify ATP anti-phishing policies: a new policy
 
     To edit an existing entry, select the protected user in the list.
 
+       **Note**: In the Security & Compliance Center, you can enter a maximum of 60 users. In Exchange Online PowerShell, you can enter a maximum of 60 users.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-atp-anti-phishing-policies.md]:
```suggestion
  > [!NOTE] 
 > In the Security & Compliance Center and in Exchange Online PowerShell you can enter a maximum of 60 users.
```

File: microsoft-365/security/office-365-security/configure-atp-anti-phishing-policies.md [hunk 2]
```diff
@@ -166,12 +166,14 @@ Use the following procedures to modify ATP anti-phishing policies: a new policy
...
+       
    - **Add domains to protect**: Configure one or both of the following settings:
 
      - **Automatically include the domains I own**: The default value is **Off**. To turn it on, slide the toggle to **On**.
      - **Include custom domains**: The default value is **Off**. To turn it on, slide the toggle to **On**, and in the **Add domains** box, enter the domain name (for example, contoso.com), press ENTER, and repeat as necessary.
 
-       **Note**: In the Security & Compliance Center, you can enter a maximum of 20 domains. In Exchange Online PowerShell, you can enter a maximum of 50 domains.
+       **Note**: In the Security & Compliance Center, you can enter a maximum of 50 domains. In Exchange Online PowerShell, you can enter a maximum of 50 domains.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-atp-anti-phishing-policies.md]:
```suggestion
  > [!NOTE]
  > In the Security & Compliance Center and in Exchange Online PowerShell you can enter a maximum of 50 domains.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@urgandhi Please implement the suggested changes. Thanks.

**Comment by urgandhi** [comment]
> @urgandhi Please implement the suggested changes. Thanks.

Done

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrisda Please merge this PR. Thanks.

**Pull request merged**